com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/graph.py

```diff
@@ -157,8 +165,7 @@ def _addOperatorFunction(self, function):
             # pickle format is binary; base64 encode so it is json serializable 
             self.params["pyCallable"] = base64.b64encode(pickle.dumps(function)).decode("ascii")
 
-        self.params["pyModule"] = function.__module__
-                    
+        self.params["pyModule"] = streamsx.topology.dependency.get_module_name(function)          
```

Comment:
Added a comment.  Also pulled out the change since we are no longer supporting functions from the `__main__` module as input to operations in this pull request.